pkg/api/mlflow/dao/repositories/namespace_cached.go

```diff
@@ -107,6 +107,10 @@ func (r NamespaceCachedRepository) Update(ctx context.Context, namespace *models
 func (r NamespaceCachedRepository) GetByCode(
 	ctx context.Context, code string,
 ) (*models.Namespace, error) {
+	result, ok := r.cache.Get(code)
+	if ok {
+		return &result, nil
+	}
```

Comment:
Please add a blank line after this (like it was before I mistakenly removed this piece of code)

Reply:
ok